### DISCUSSION TITLE
How to use allow_llm_to_see_data to reply in other languages?

### DISCUSSION THREAD

**Discussion by ShawshankLin** [vanna-ai/vanna#458] [General]
How to use allow_llm_to_see_data to reply in other languages

**Comment by zainhoda** [comment]
https://github.com/vanna-ai/vanna/blob/a72b842d420cf1fa061e5f97d45ea08051651ebb/src/vanna/base/base.py#L81

If you set `language` in the config, it should use that to set the response language
